### PR TITLE
modules: lpe: small updates and fixes

### PR DESCRIPTION
**Pull request by TSC21**
This PR:
1. removes unused subscriber to manual_control_setpoint;
2. correctly publishes the innovation covariance to `ekf2_innovations`;
3. updates `_sensorTimeout` and `_sensorFault` to handle support of more sensor sources (ex: https://github.com/PX4/Firmware/pull/8160 - landing targets)

@ndepal does this fit your needs?

@jgoppert can you review, please?

### CODE REVIEW

File: src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp [hunk 1]
```diff
@@ -369,20 +373,20 @@ class BlockLocalPositionEstimator : public control::SuperBlock
 	bool _lastArmedState;
 
 	// masks
-	uint8_t _sensorTimeout;
-	uint8_t _sensorFault;
+	uint16_t _sensorTimeout;
```
Comment (ndepal) [inline src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp]: You need to initialize with 65535 in the constructor.

Comment (TSC21) [reply src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp]: You are right. Sorry

**Comment by TSC21** [comment]
Jenkins test this please.

File: src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp (hunk 1 above)
Comment (TSC21) [reply src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp]: @ndepal fixed

**Comment by TSC21** [comment]
> I think 1 << 8 overflows the 8 bit field.

That's why it was changed to `uint16_t` - https://github.com/PX4/Firmware/pull/8226/files#diff-a072a0a21e8690b891c1be60260d1bc7R376

**Pull request merged**